src/dex/definitions/sundaeswap-v3/order.ts

```diff
@@ -0,0 +1,82 @@
+import { DatumParameterKey } from '@app/constants';
+
+export default {
+  constructor: 0,
+  fields: [
+    {
+      bytes: DatumParameterKey.PoolIdentifier
```

Comment:
The datum has changed, so this needs to be wrapped in a constructor 0.

I'll share some helpful references for our datums in a bit!

Reply:
https://github.com/SundaeSwap-finance/sundae-sdk/blob/main/packages/core/src/DatumBuilders/DatumBuilder.Lucid.V3.class.ts

https://github.com/SundaeSwap-finance/sundae-sdk/blob/main/packages/core/src/DatumBuilders/contracts/contracts.v3.ts

https://github.com/SundaeSwap-finance/sundae-contracts/blob/main/lib/types/order.ak#L8-L31

Reply:
Fixed in 9a31ee95cb7024e03f3da20e4e91deea8cb301bd